packages/protocol/scripts/bash/revoke_contracts.sh

```diff
@@ -0,0 +1,68 @@
+#! /usr/bin/env bash
+set -euo pipefail
+rejects=()
+
+if [ "$#" -ne 1 ]; then
+  echo "Expected usage: ./revoke_contracts.sh contracts.txt"
+  echo "contracts.txt should include contracts you'd like to revoke and refund, one address per line"
+  exit 1
+fi
+
+for contract in $(<"$1"); do
+  echo "Contract $contract"
+
+  # Check that account has been created
+  accountExists=1
+  celocli lockedgold:show "$contract" > /dev/null 2>&1 || accountExists=0
+  if [ $accountExists -eq 0 ]; then
+    echo "Account not created, rejecting $contract"
+    rejects+=( $contract )
+    continue
+  fi
+
+  # Check that CELO has been unlocked
+  lockedgold=$(celocli lockedgold:show "$contract")
+  total=$(echo "$lockedgold" | grep -o "total: [0-9]*" | cut -f2 -d : | tr -d '[:space:]')
```

Comment:
Assuming this doesn't include pending withdrawals

Reply:
Yep this is just to verify that there is no gold locked.

While the $total here is passed in to `releasegold:locked-gold.withdraw`, the `withdraw` fn actually just withdrawals all available pending withdrawals, regardless of value. FWIW, this is exactly how `lockedgold:withdraw` works, but I guess I forgot to make `value` optional for the `action.withdraw` on `releasegold:locked-gold`

This has no real impact on functionality, but is unclear, so should be more clear. I'll create a tech debt issue